ClinGen/clincoded
# Connects to #1380. New help docs.

**Technical notes:**
1. Added new help documents for GCI, VCI and MonDO search. Updated symlinks.
2. Added `meta.gdm.disease.term` and `meta.interpretation.disease.term` columns in the `curatorHistory.json` schema for history item consisting of diseases displayed on the Dashboard page.
3. Updated the `history.py` upgrade script to handle disease UUIDs migration, specially targeting the `['disease']` property nested in `['meta']['interpretation']` which may not always have a disease associated.

**Steps to test:**
1. Go to the `^/create-gene-disease/` page and click on the **Disease+** button.
2. In the displayed modal, expect to see the **MonDO Search Help** link at the top of the modal.
3. Click on this link and expect to see a new window/tab displaying the help doc.
4. From the **Help** dropdown in the top nav, select either the **Gene Curation** or **Variant Curation** option. Expect to see the updated docs in a new window/tab.
5. Create a new GDM and return to the Dashboard page. Expect to see the disease term being displayed in the **recent history** item.
6. Start a new variant interpretation and associate it with a disease. Return to the Dashboard page and expect to see the disease term being displayed (instead of "undefined").
